mailpoet/lib/Entities/ScheduledTaskEntity.php

```diff
@@ -21,6 +21,7 @@ class ScheduledTaskEntity {
   const STATUS_PAUSED = 'paused';
   const STATUS_INVALID = 'invalid';
   const VIRTUAL_STATUS_RUNNING = 'running'; // For historical reasons this is stored as null in DB
+  const TYPE_SENDING = 'sending';
```

Comment:
Other scheduled task type constants are declared in related worker classes (e.g. [#](https://github.com/mailpoet/mailpoet/blob/5373c811b2525de7a586e850a2c8fa9cfc04c168/mailpoet/lib/Cron/Workers/Bounce.php#L24), [#](https://github.com/mailpoet/mailpoet/blob/af396021db8ee4da756409d53c731b08c93c6926/mailpoet/lib/Cron/Workers/InactiveSubscribers.php#L12)).
So I would suggest declaring this type constant in [the SendingQueue worker](https://github.com/mailpoet/mailpoet/blob/8d4799b050bf058b8bb73ceb056c5f02ca0a33ec/mailpoet/lib/Cron/Workers/SendingQueue/SendingQueue.php#L32).
Also please note that this constant is already defined in https://github.com/mailpoet/mailpoet/blob/9b71a7804cd7d3c8724c9ac877cc62def4d3a697/mailpoet/lib/Tasks/Sending.php#L26
So maybe we could remove it from there.

Reply:
Thank you for this tip. I agree!